rust/sbd-e2e-crypto-client/src/lib.rs

```diff
@@ -164,10 +167,13 @@ impl SbdClientCrypto {
             return Err(Error::other("message too long"));
         }
 
+        // get or create an encryptor, returns a list of messages to send
         let enc = self.inner.lock().unwrap().enc(pk, Some(msg))?;
 
         {
             let client = self.client.lock().await;
+
+            // send the encrypted messages to send
```

Comment:
Something like this?

```suggestion
            // send the pending encrypted messages
```

Reply:
haha : )